docs/diagrams/ModelClassDiagram.puml

```diff
@@ -7,14 +7,17 @@ skinparam classBackgroundColor MODEL_COLOR
 Package Model <<Rectangle>>{
 Class "<<interface>>\nReadOnlyAddressBook" as ReadOnlyAddressBook
 Class "<<interface>>\nReadOnlyUserPrefs" as ReadOnlyUserPrefs
+Class "<<interface>>\nReadOnlyTaskList" as ReadOnlyTaskList
 Class "<<interface>>\nModel" as Model
 Class AddressBook
 Class ModelManager
 Class UserPrefs
+Class TaskList
+Class Task
```

Comment:
Task should point to tag with a `*` and should point to another class `Link`

Reply:
👌

Reply:
> Other than a missing association and class in `ModelClassDiagram`, everything else LGTM.
> 
> If able to, could tweak `StorageClassDiagram` as its hard to see the multiplicity.

I get what you mean for StorageClass, shifted AddressBookStorage and TaskListStorage but can't figure out a way to layout the zones in a way that is nice and can still see the multiplicity. Anyone able to fix this?